### ISSUE TITLE
Quest: Hampering Their Escape

### ISSUE THREAD

**Issue by Ergates82** [dalaranwow/dalaran-wow#5087]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The quest Hampering Their Escape in Borean Tundra, given by Etaruk, and part of a quest chain involves collecting explosives from Gnome workers, and then tossing these explosives at three floating platforms. When I right click on the grenade, it has no trajectory, and does not fly up to the platform, rendering the quest uncompletable. it may be possible to complete on a flying mount, but I have no means of testing this.

**Expected behaviour**: Tell us what should happen instead

The grenade is supposed to be thrown upward in a kind of arc, and land upon the floating platform, 'blowing it up' and completing quest requirements. Instead it simply goes off in my hand, and is not thrown

**Steps to reproduce the problem**:

1. Visit Etaruk in Coldrock Quarry, Borean Tundra, and complete the quest 'A Visit to the Curator'
2. Hand that quest in, and then pick up 'Hampering their Escape'
3. Kill 3 gnome workers, loot their grenades
4. Stand under a floating platform, or slightly above one on the cliffs and toss a grenade.

**Include proofs for this behaviour**

http://www.wowhead.com/quest=11617/hampering-their-escape
https://www.youtube.com/watch?v=tsnlWukay_E

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Ergates82.
I just did this quest on PTR and there was a projectile:
![wowscrnshot_112217_134939](https://user-images.githubusercontent.com/6797611/33126201-56af7224-cf8c-11e7-95f4-1ffec9124f30.jpg)

and a quest credit after that. Currently this area doesn't look good but it was reworked in #4554

Note: Try to stay as close as possible to the platforms.
